classes/Project/ProjectFactory.js

```diff
@@ -181,6 +184,284 @@ export default class ProjectFactory {
+      if (withAnnotations) {
+        return await this.clonePagesWithAnnotations(layer, page, copiedProject, database)
+      }
+      return await this.clonePageWithoutAnnotations(page, copiedProject, database)
+    }))
+    return newPages
+  }
+
+  static async clonePageWithoutAnnotations(page, copiedProject, database) {
+    return {
+      id: `${process.env.SERVERURL}project/${copiedProject._id}/page/${database.reserveId()}`,
+      label: page.label,
+      target: page.target
+    }
+  }
+
+  static async clonePagesWithAnnotations(layer, page, copiedProject, database) {
+    if(!page.id.startsWith(process.env.RERUMIDPREFIX)) {
+      return {
+        id: `${process.env.SERVERURL}project/${copiedProject._id}/page/${database.reserveId()}`,
+        label: page.label,
+        target: page.target
+      }
+    }
+    else {
+      return await fetch(page.id)
+      .then(response => response.json())
+      .then(async pageData => {
+        const newPage = new Page(layer.id, {
+          id: `${process.env.SERVERURL}project/${copiedProject._id}/page/${database.reserveId()}`,
+          label: page.label,
+          target: page.target,
+          items: await Promise.all(pageData.items.map(async item => {
+            return await fetch(item.id)
+            .then(response => response.json())
+            .then(async itemData => {
+              const newItem = new Line({
+                id: `${process.env.SERVERURL}project/${copiedProject._id}/line/${database.reserveId()}`,
+                target: itemData.target,
+                body: itemData.body,
+                motivation: itemData.motivation,
+                label: itemData.label,
+                type: itemData.type
+              })
+              return await newItem.update()
+            })
+          }))
+        })
+        return await newPage.update()
+      })
+    }
+  }
+
+  static async copyProject(projectId, creator) {
+    if (!projectId) {
+      throw {
+        status: 400,
+        message: "No project ID provided"
+      }
+    }
+
+    const project = await ProjectFactory.loadAsUser(projectId, creator)
+    if (!project) {
+      throw {
+        status: 404,
+        message: "Project not found"
+      }
+    }
+
+    let copiedProject = this.copiedProjectConfig(project, database, creator)
+    await this.cloneLayers(project, copiedProject, database, true)
+    copiedProject._lastModified = copiedProject.layers[0]?.pages[0]?.id.split('/').pop()
+    const copiedGroup = await this.cloneGroup(project, creator, { 'Group Members': true })
+    await this.cloneHotkeys(project._id, copiedProject._id)
+    return await new Project().create({ ...copiedProject, creator, group: copiedGroup._id })
+  }
+
+  static async cloneWithoutAnnotations(projectId, creator) {
+    if (!projectId) {
+      throw {
+        status: 400,
+        message: "No project ID provided"
+      }
+    }
+
+    const project = await ProjectFactory.loadAsUser(projectId, creator)
```

Comment:
Definitely don't loadAsUser() here. This should be a copy of the database object, not the projection for the User.